src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -411,6 +411,14 @@
     )
     private String respondentUserId;
 
+
+    @CCD(
+        label = "Date when the application was issued",
+        access = {DefaultAccess.class}
```

Comment:
Small detail but we might want a CaseworkerAccess class, because this should be CRU for caseworker and legal advisors but READ for everyone else

Reply:
done as part of https://github.com/hmcts/nfdiv-case-api/pull/374/commits/e53ee57a1416b04d00112711a02094f101b39319